.github/workflows/atlantis-base.yml

```diff
@@ -47,9 +47,10 @@ jobs:
     - name: Build and push atlantis-base:${{env.TODAY}} image
       uses: docker/build-push-action@v3
       with:
+        cache-from: type=gha
+        cache-to: type=gha,mode=max
         context: docker-base
         platforms: linux/arm64/v8,linux/amd64,linux/arm/v7
-        pull: true
```

Comment:
Can we do the pull anyway? Does it hurt to pull?

Reply:
We can do it, it causes an extra request to Docker Hub or wherever the base image is hosted. In the case of the `atlantis-base` Docker image, I don't assume it ever will be updated after it has been pushed, so it doesn't make sense to add `pull: true` for that.

The other base image used that I can see is the `golang` one. For that I think it's probably better to pin the exact Go version you want, which already is done here:
https://github.com/runatlantis/atlantis/blob/e60dd843f48124e20105a695c1fbd404fc3d8914/Dockerfile#L2

It is however not done here:
https://github.com/runatlantis/atlantis/blob/e60dd843f48124e20105a695c1fbd404fc3d8914/testing/Dockerfile#L1

Reply:
ah then perhaps we can update the testimg image to be the same FROM as the main image and then keep the current prs behavior of removing the pull functionality

Reply:
@Tenzer

Reply:
Done now. I didn't switch it to use Alpine as that would be a bigger change and I'm not sure if that's desired.